src/views/virtualmachinesinstance/details/tabs/disks/VirtualMachinesInstancePageDisksTab.scss

```diff
@@ -0,0 +1,5 @@
+.VirtualMachinesInstancePageDisksTab {
+  td#drive {
```

Comment:
Why this?

Reply:
This was already there, I just renamed the file and removed unnecessary parts of it. However, this wasn't unnecessary, it does the following: disks -> Disks. Not sure why it was done this way, but I've better left it like that. Or we could ask Matan why...